### PR TITLE
Fix NEX-1258 - Language not returned in shared stimulus

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/NEX-1258

**Description**

Change param sent to delete API to the old way because that endpoint don't suffer any change

**AC**
- [x] Use old properties for call delete endpoint

### CODE REVIEW

File: views/js/controller/actions.js [hunk 1]
```diff
@@ -154,7 +154,10 @@ define([
                                 if (actionContext.tree) {
                                     $(actionContext.tree).trigger('removenode.taotree', [
                                         {
-                                            id: actionContext.uri || actionContext.classUri
+                                            uri: actionContext.uri,
```
Comment (boajer) [inline views/js/controller/actions.js:157]: I assume payload should be adjusted here
Screenshot 2020-06-19 at 08 38 10

Comment (boajer) [inline views/js/controller/actions.js:157]: I think payload should be fixed here
Screenshot 2020-06-19 at 08 45 33

**Comment by gabrielfs7** [comment]
@habemuscode The task you took is a BE and related to language: https://oat-sa.atlassian.net/browse/NEX-1227

- [ ] Please update the PR and link to the specific task :)

File: views/js/controller/actions.js [hunk 2]
```diff
@@ -154,7 +157,7 @@ define([
                                 if (actionContext.tree) {
                                     $(actionContext.tree).trigger('removenode.taotree', [
                                         {
-                                            id: actionContext.uri || actionContext.classUri
+                                            id : actionContext.uri || actionContext.classUri
```
Comment (bziondik) [inline views/js/controller/actions.js:160]:
```suggestion
                                            id : actionContext.uri || actionContext.classUri;
```

Comment (boajer) [reply views/js/controller/actions.js:160]: Are you sure?

Comment (bziondik) [reply views/js/controller/actions.js:160]: sorry :)

**Pull request merged**